### ISSUE TITLE
paramiko error

### ISSUE THREAD

**Issue by gregschap123** [git-keeper/git-keeper#138]
** Written by Ben using Greg's account...**

Greg updated to Python3.7.1, and had to re-run the install process.  `gkeep` works, but any command that uses ssh outputs the following warning:

```
$ gkeep query classes
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/paramiko/ecdsakey.py:164: CryptographyDeprecationWarning: Support for unsafe construction of public numbers from encoded data will be removed in a future version. Please use EllipticCurvePublicKey.from_encoded_point
  self.ecdsa_curve.curve_class(), pointinfo
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:39: CryptographyDeprecationWarning: encode_point has been deprecated on EllipticCurvePublicNumbers and will be removed in a future version. Please use EllipticCurvePublicKey.public_bytes to obtain both compressed and uncompressed point encoding.
  m.add_string(self.Q_C.public_numbers().encode_point())
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:96: CryptographyDeprecationWarning: Support for unsafe construction of public numbers from encoded data will be removed in a future version. Please use EllipticCurvePublicKey.from_encoded_point
  self.curve, Q_S_bytes
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/paramiko/kex_ecdh_nist.py:111: CryptographyDeprecationWarning: encode_point has been deprecated on EllipticCurvePublicNumbers and will be removed in a future version. Please use EllipticCurvePublicKey.public_bytes to obtain both compressed and uncompressed point encoding.
  hm.add_string(self.Q_C.public_numbers().encode_point())
120f18a
120f18c
120f18la
222s19a
234s19a
```

**Comment by sommern** [comment]
Looks like this is happening due to the `cryptography` package:
https://github.com/paramiko/paramiko/issues/1369

This is a dependency for which we neglected to fix a version number in `requrements.txt`. Probably the best action to take would be to fix the `cryptography` version to 2.4.2 until this settles down.

**Comment by sommern** [comment]
Fixed in #172
